src/Makefile

```diff
@@ -147,6 +147,7 @@ $(BUILD)/micropython.js: $(OBJ) jshal.js simulator-js
 
 simulator-js:
 	npx esbuild ./simulator.ts --bundle --outfile=$(BUILD)/simulator.js --loader:.svg=text
+	npx esbuild --define:process.env.version=$$(cat ../package.json | jq .version) ./sw.ts --bundle --outfile=$(BUILD)/sw.js
```

Comment:
This gives us a version we can use in `sw.ts` via `process.env.version` which is replaced at build time.

Maybe using `process.env.version` in the service worker is misleading, but it doesn't raise type issues about a value that doesn't exist.

It breaks the Cloudflare build because `jq is not found`. Is there a nice way that doesn't use jq?

Reply:
Changed this to not use jq. It's not as nice. Better way? [1524cb8](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/115/commits/1524cb88f6eb6b52bd1d03cd2f7b241b3423d08b)

Reply:
Use node [1331423](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/115/commits/1331423d2b0b1050e4b06908560d8571a12f65e6)